### DISCUSSION TITLE
Question: How to allow people to access certain plots in a town

### DISCUSSION THREAD

**Discussion by TheIcyHot** [TownyAdvanced/Towny#4505] [Q&A] [answered]
Ok, so let's set the scene. Let's say there is a person in my town named Joe, and let's say I am the mayor and I want to make a storage area for myself and Joe. So I claim the land and designate it as my plot, is there a way that I can give Joe access to that SPECIFIC plot without adding him as a friend while he keeps his own plot and land that he has claimed in my town. Basically, is there a way to give specific people in your town access to certain plots or chunks of your town without promoting them or anything else? While you maintain ownership of that plot/chunk. I'm sorry if I'm doing this wrong this is my first post on here lol.

**Comment by LlmDl** [comment]
Add him as a friend and only allow the friend plot permissions on that plot?

**Comment by TheIcyHot** [comment]
> Add him as a friend and only allow the friend plot permissions on that plot?

I don't want to turn off friend access for all my other plots though.

**Comment by LlmDl** [comment, marked as answer]
You turn on the friend access in one plot using /plot set perm friend on

**Comment by TheIcyHot** [comment]
> You turn on the friend access in one plot using /plot set perm friend on

I am aware of that, but I would have to deactivate friend access for all my other plots that I don't want them in and I don't want to do that.

**Comment by LlmDl** [comment]
You could use a chest-protection plugin that has a friends list per-storage. I believe there might be a ticket already suggesting a townblock-specific friendslist of sorts.
